tofhir-server/src/main/scala/io/tofhir/server/service/job/JobFolderRepository.scala

```diff
@@ -191,6 +191,10 @@ override def getJob(projectId: String, id: String): Future[Option[FhirMappingJob
     }
     var directories = Seq.empty[File]
     directories = folder.listFiles.filter(_.isDirectory).toSeq
+    if (directories.isEmpty) {
+      throw new RuntimeException("There is no project folder for job in the repository")
```

Comment:
Let's remove this commit because it prevents tofhir-server from starting when there is no project. We'll implement the correct logic in a new commit.